elements/pfe-icon/src/pfe-icon.scss

```
@@ -113,7 +113,9 @@ $sizing: (
 
 @each $level in (critical, important, moderate, success, info, default) {
   :host([color="#{$level}"]) {
-    --pfe-icon--color: #{pfe-var(feedback--#{$level})};
+    filter feFlood {
+      flood-color: #{pfe-var(feedback--#{$level})};
```

To prevent regressions, I'd recommend:

```suggestion
      flood-color: pfe-local(color, $fallback: pfe-var(feedback--#{$level}));
```

Also just a note that you don't need interpolation on functions when assigning them to css properties, only when assigning them to custom properties.